webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
```

Comment:
could we use [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)?